<launch>

	<arg name="dt"          default="3"   />
	<arg name="sigma_xy"    default="0.1" />
	<arg name="sigma_angle" default="0.25"/>
	<arg name="linear_th"   default="0.05"/>
	<arg name="angular_th"  default="0.1" />

	<!-- amcl spreader -->
	<node pkg="uchile_navigation_utils" type="amcl_spreader" name="amcl_spreader" respawn="false" output="screen" ns="/bender/nav/">
		<param name="dt"          value="$(arg dt)"         />
		<param name="sigma_xy"    value="$(arg sigma_xy)"   />
		<param name="sigma_angle" value="$(arg sigma_angle)"/>
		<param name="linear_th" value="$(arg linear_th)"/>
		<param name="angular_th" value="$(arg angular_th)"/>
		<remap from="amcl_spreader/cmd_vel" to="/bender/nav/base/cmd_vel"/>
	</node>

</launch>
